### PR TITLE
Removed `output` argument from ERT jobs (+ ensured output folder exists)

### PR DESCRIPTION
**Pull request by lindjoha**
Solving #694.

Removing OUTPUT argument from ert config jobs, but not from terminal scripts. That way the default output file will be used when running from ert.

Also added a line in both ert jobs to create the output folder if it doesn't exist.

### CODE REVIEW

File: webviz_subsurface/ert_jobs/config_jobs/WELL_CONNECTION_STATUS [hunk 1]
```diff
@@ -1,5 +1,6 @@
 EXECUTABLE well_connection_status
 
-ARGLIST <INPUT> <OUTPUT>
+ARGLIST <INPUT>
 
-MIN_ARG 2
+MIN_ARG 1
+MAX_ARG 1
```
Comment (anders-kiaer) [inline webviz_subsurface/ert_jobs/config_jobs/WELL_CONNECTION_STATUS:6]: Missing newline.

File: webviz_subsurface/ert_jobs/well_connection_status.py [hunk 2]
```diff
@@ -117,6 +117,10 @@ def main() -> None:
     args = parser.parse_args()
 
     df = _extract_well_connection_status(args.input)
+
+    # Create the output folder if it doen't exist
+    args.output.mkdir(parents=True, exist_ok=True)
```
Comment (anders-kiaer) [inline webviz_subsurface/ert_jobs/well_connection_status.py:122]: This will maybe also create a folder called `well_connection_status.parquet`?

**Review by anders-kiaer — CHANGES_REQUESTED** [review]
Also change the `test_smry2arrow` test?

File: webviz_subsurface/ert_jobs/config_jobs/WELL_CONNECTION_STATUS (hunk 1 above)
Comment (lindjoha) [reply webviz_subsurface/ert_jobs/config_jobs/WELL_CONNECTION_STATUS]: fixed

File: webviz_subsurface/ert_jobs/well_connection_status.py (hunk 2 above)
Comment (lindjoha) [reply webviz_subsurface/ert_jobs/well_connection_status.py:123]: I will check.

Comment (lindjoha) [reply webviz_subsurface/ert_jobs/well_connection_status.py:123]: yes it does

**Review by anders-kiaer — APPROVED** [review]

**Pull request merged**